third_party_release

```diff
@@ -5,6 +5,6 @@ benchmark=v1.5.3
 googletest=release-1.8.0-2523-ga6dfd3ac
 ms-gsl=v1.0.0-393-g6f45293
 nlohmann-json=v3.9.1
-opentelemetry-proto=v0.11.0
+opentelemetry-proto=v0.11.0-16-g3c2915c
```

Comment:
Should here be 0.15.0 as well?

Reply:
It's strange this is what `git submodule` returns.
There were some breaking changes, I need to fix them for the upgrade.

Reply:
> There were some breaking changes, I need to fix them for the upgrade.

Is it related to proto file changes for logs sdk?

Reply:
yes, it should be related to log.